.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+       - image: circleci/ruby:2.5.1-node-browsers
+
+    steps:
+      - checkout
+
+      # install dependencies
+      - run:
+          name: Install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+
+      # run tests
+      - run:
+          name: Run tests
+          command: |
+            bundle exec rspec --color --require spec_helper spec --format progress
```

Comment:
Could we have a `bundle exec rubocop` run step?

Reply:
(Before the `rspec` step.)

Reply:
added in a next commit